scripts/benchmark_runner_linux.py

```diff
@@ -0,0 +1,173 @@
+"""Run a benchmark and return the results."""
+import argparse
+
+import subprocess
+import time
+import os
+import urllib.request
+import benchmark_utils as utils
+
+
+def create_extract_destinations(path):
+    """ Create the extract destination directories if they do not exist"""
+    if not os.path.exists(path):
+        os.mkdir(path)
+    os.chdir(path)
+    if not os.path.exists("base"):
+        os.mkdir("base")
+    if not os.path.exists("daily"):
+        os.mkdir("daily")
+
+
+def download_file(url, filename):
+    """ Download file from url and save it to filename"""
+    with urllib.request.urlopen(url) as response, open(filename, 'wb') as out_file:
+        data = response.read()
+        out_file.write(data)
+
+
+def download_and_extract_dotnet_sdk(version_url, extract_path):
+    """ Download and extract the dotnet sdk"""
+
+    tar_gz_file = "dotnet-sdk.tar.gz"
+    download_file(version_url, tar_gz_file)
+
+    # Extract the tar.gz file
+    # extract_command = f"tar -xzf {tar_gz_file} -C {extract_path}"
+    subprocess.run(["tar", "-xzf", tar_gz_file,
```

Comment:
Would it be possible to not use `tar` here, just pure python? (It makes it more portable and easier to test locally)

Reply:
Yes! Sure ✅